### DISCUSSION TITLE
Can we identify a client by ASN?

### DISCUSSION THREAD

**Discussion by flinhong** [AdguardTeam/AdGuardHome#2867] [Ideas]
Hi, is it useful to identify clients by their ASN?

I used DoT on my Android phone and almost using mobile data all day long. However, its IP changes a lot, can we identify this kind of client by combining ASN and requests type of DoT?

**Comment by ameshkov** [comment]
Currently, you can only use CIDRs for that.

There is also a feature request that is relevant: https://github.com/AdguardTeam/AdGuardHome/issues/1487
